wger/settings.py

```diff
@@ -0,0 +1,60 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+from wger.settings_global import *
+import os
+# Use 'DEBUG = True' to get more details for server errors
+DEBUG = True
+TEMPLATES[0]['OPTIONS']['debug'] = True # noqa
+ROOT_DIR = os.path.dirname(os.path.abspath(__file__))
+
+ADMINS = (
+    ('Your name', 'your_email@example.com'),
+)
+MANAGERS = ADMINS
```

Comment:
Good work getting the app working on heroku. Please consider moving the ADMINS setting to env and also provide .env.sample for us setting up in our machines.

Reply:
Changes made